src-docs/src/views/header/header_elastic_pattern.js

```diff
@@ -283,7 +283,9 @@ export default ({ theme }) => {
         sections={[
           {
             items: [
-              <OuiHeaderLogo iconType="logoOpenSearch" href="">
+              <OuiHeaderLogo
+                iconType="logoOpenSearch"
+                href="https://oui.opensearch.org/latest/">
```

Comment:
Hover over the logo

<img width="600" alt="Screenshot 2023-04-18 at 09 15 48" src="https://user-images.githubusercontent.com/17855243/232661884-aaeb6ed9-15c0-451d-84bd-fffd11a8aa97.png">